spec/sections_spec.rb

```diff
@@ -22,6 +28,24 @@ class Page < SitePrism::Page
     it { is_expected.to respond_to(:has_single_section?) }
   end
 
+  describe '.section with class and block' do
+    before do
+      expect(subject).to receive(:find_first).and_return(:element)
+    end
+
+    it 'should be an instance of provided section class' do
```

Comment:
I'm confused by what this test is verifying?

Reply:
So, it is creating an anonymous class [here](https://github.com/natritmeyer/site_prism/pull/250/files#diff-0142fe7beaa583d4ce8030ab2a152817R242) with parent SingleSection and I want to check it